cicd/42c-conf-examples/mapping-no-discovery/42c-conf.yaml

```diff
@@ -2,18 +2,15 @@
 # only audit the API files you have specifically mapped to existing 
 # APIs in 42Crunch Platform.
 #
-# Discovery phase is enabled by default, otherwise the CI/CD integration
-# plugin would not find any API files to audit. However, if you map the
-# filenames of the API definitions that you want to include to
-# the API UUIDs in the platform, you can exclude any other files.
-#
-# When the plugin starts, it begins with cleaning the 
-# API collection you have defined it to use. That is why the mapping 
-# must be done to another API collection.
+# When the CI/CD plugin starts, it creates an API collection
```

Comment:
"When the CI/CD plugin starts, it creates an API collection using the source control repository name and branch name." --> "When the CI/CD plugin starts, it creates an API collection in 42Crunch Platform using the name of the repository and branch directly from your source control."